danger/instant-checks.rb

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
```

Comment:
:warning: The year in this license header is outdated, time to update!





<p align="right" data-meta="generated_by_instant-checks">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>